sdk/security/keyvault/azkeys/client_test.go

```diff
@@ -159,6 +159,16 @@ func TestCRUD(t *testing.T) {
 				require.Equal(t, createResp.Key.KID.Version(), getResp.Key.KID.Version())
 				testSerde(t, &getResp.KeyBundle)
 
+				if mhsm {
+					getAttResp, err := client.GetKeyAttestation(context.Background(), keyName, "", nil)
+					require.NoError(t, err)
+					require.Equal(t, createResp.Key.KID.Name(), getAttResp.Key.KID.Name())
+					require.Equal(t, createResp.Key.KID.Version(), getAttResp.Key.KID.Version())
+					require.NotNil(t, getAttResp.Attributes.Attestation.CertificatePEMFile)
+					require.NotNil(t, getAttResp.Attributes.Attestation.PrivateKeyAttestation)
```

Comment:
These are `[]byte`; do you want `NotEmpty`?

Reply:
Good catch, thanks!